Analyzer/interface/CommonFunction.h

```diff
@@ -1573,8 +1573,10 @@ reco::DeDxData computedEdx(const int& run_number,
       std::vector<int> amplitudes = convert(cluster->amplitudes());
       std::vector<int> amplitudesPrim = CrossTalkInv(amplitudes,0.10,0.04,true);
 
-      //if (useClusterCleaning && !clusterCleaning(amplitudes, crossTalkInvAlgo)) continue;
-      if (useClusterCleaning && !clusterCleaning(amplitudes, 1)) continue;
+      // why is this hardcoded now?
```

Comment:
So this was explained by Caroline, I just forgot to add it, I'll do it next time